frontend/src/v5/ui/themes/theme.ts

```diff
@@ -1030,8 +1030,9 @@ export const theme = createTheme({
 			styleOverrides: {
 				contained: {
 					position: 'absolute',
-					bottom: -18,
 					margin: 0,
+					top: 55,
```

Comment:
This causes in large inputs, such as custom tickets description boxes, pins, viewpoints, and images
![image](https://user-images.githubusercontent.com/62481250/212671143-cc11a74e-9845-4ea3-9b39-9e531ab96ca3.png)